src/streaming/controllers/PlaybackController.js

```diff
@@ -663,7 +663,7 @@ function PlaybackController() {
                     ranges = bufferedRange[streamInfo.id].video;
                 }
                 if (checkTimeInRanges(earliestTime, ranges)) {
-                    if (!isSeeking() && !compatibleWithPreviousStream) {
+                    if (!isSeeking() && !compatibleWithPreviousStream && earliestTime !== 0) {
```

Comment:
also needed line 676 (current stream has only audio or only video content case) I guess

Reply:
not really because the first test for audio or video only is : 

if (commonEarliestTime[streamInfo.id][type]) 

So, in this case, the value is 0, test is false and no useless seek is done.

Reply:
yeah my bad you're right this is not needed for 2nd case. Thanks!